Makefile

```diff
@@ -86,7 +86,7 @@ endif
 
 # Run all tests
 .PHONY: test
-test: test-dep validate test-unit
+test: test-dep validate test-unit test-jsonschema-generation
```

Comment:
IMO this should not be included.

This suddenly uses a Docker container. 

Similar to how we have integration tests / unit tests / cmd tests separated into their separate commands.

Reply:
Is it not safe to assume that the kedge developers have docker installed locally?

Reply:
no, it's not safe, and tests should be ran separately... testing json-schema-generation should be separate just like test-unit, test-ci, test-cmd, etc.

Reply:
done changes as requested!